routes/stripe.js

```diff
@@ -15,7 +15,7 @@ const {
 
 router.route('/authorize').get(authorizeStripeUser)
 router.route('/finalize').post(finalizeStripeUser)
-router.route('/isauthorized').post(isAuthorizedUser)
+router.route('/isauthorized').get(isAuthorizedUser)
```

Comment:
Since the `/api/stripe/isauthorized` endpoint is changing from `POST` to `GET`, merging this PR would require an additional change to `Karpule-App`. We need to ensure that we reflect this change once this PR is merged.